data/zilliqa/assets/zil1kwfu3x9n6fsuxc4ynp72uk5rxge25enw7zsf9z/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "Score",
+	"symbol": "SCO",
+	"web": "https://www.uffsports.com",
+	"supply": "100000000",
+	"email": "info@uffsports.com",
+	"whitepaper": "https://docs.google.com/document/d/1ObJ5CxOs8k9TJssLUPdWLLM0EmZngpXY/edit",
+	"publicTeam": true,
```

Comment:
Not saying any team on the website. Is there public linkedin profiles?

Reply:
Hi. Only a few of us have a LinkedIn account. 
I don't think we have any web page about the team

Reply:
Could you post the available linkedings in the thread?

Reply:
https://www.linkedin.com/in/andrea-ellis-08a87143/

Reply:
https://www.linkedin.com/in/dave-kleinsasser-869998195/

Reply:
I think that most of the linkedin profiles don't even mention UFF Sports lol
We used to have a team page on our old website

Reply:
https://www.linkedin.com/in/larry-fisher-6110aa8b/

Reply:
https://www.linkedin.com/in/jason-kleinsasser-08312000/